src/services/ConfigService.jsx

```diff
@@ -0,0 +1,20 @@
+export class ConfigService {
+  constructor({ apiClient }) {
+    this.apiClient = apiClient
+  }
+
+  getEndpoint({ siteName, isHomepage, isSettings }) {
+    if (isHomepage) return `/sites/${siteName}/homepage`
+    if (isSettings) return `/sites/${siteName}/settings`
+  }
+
+  async update(apiParams, { configConfig }) {
```

Comment:
wdyt about renaming `configConfig` to something liek `endpointConfig` or something more descriptive/specific? i feel like it would help with readability/understanding!

Reply:
Yes! Good catch, haha I think I didn't look over the naming after I used the replace functionality. I refactored this in d1ec478